### PR TITLE
feat(FEC-10640): add api to get the playlist current working item index

### PR DESCRIPTION
**Pull request by yairans**
### Description of the Changes

add `current` api to the playlist manager and add `index` field to the playlist item
### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [x] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

**Review by OrenMe — DISMISSED** [review]

**Comment by OrenMe** [comment]
this refactor actually makes the code nicer :-)

File: src/common/playlist/playlist-manager.js [hunk 1]
```diff
@@ -95,8 +95,8 @@ class PlaylistManager {
   playNext(): void {
     PlaylistManager._logger.debug('playNext');
     const next = this._playlist.getNext(true);
```
Comment (Yuvalke) [inline src/common/playlist/playlist-manager.js:97]: It's not about the change but maybe you will know why doesn't it use the getter next and force to get the next one even the loop config is false?

**Review by Yuvalke — DISMISSED** [review]

File: src/common/playlist/playlist-manager.js (hunk 1 above)
Comment (yairans) [reply src/common/playlist/playlist-manager.js:97]: loop it's for the UI and for automatically next playing only. for the api loop it's always true

Comment (yairans) [reply src/common/playlist/playlist-manager.js:97]: loop it's for the UI and for automatically next playing only. for the api loop it's always true

**Pull request merged**